### ISSUE TITLE
Pushback Failure

### ISSUE THREAD

**Issue by JustDavidOk** [skiselkov/BetterPushbackC#203]
I get a pushback failure each time I try to use the plugin. When I choose where I want to go I get an error "Pushback failure, cannot find suitable tug for your aircraft". When I try to select it again it works perfectly. I'm using the mac version with the FlightFactor A320.

Log errors:
2018-02-13 14:50:47 BetterPushback[acf_outline.c:198]: Error parsing acf file: property _wing/13/_Croot not found
2018-02-13 14:50:47 BetterPushback[acf_outline.c:198]: Error parsing acf file: property _wing/15/_Croot not found
2018-02-13 14:51:07 BetterPushback[wav.c:268]: Error loading WAV file /Users/user/Desktop/X-Plane 11//Resources/plugins/BetterPushback/objects/tugs/AST-3F.tug/engine.wav: alGenBuffers failed (0xa003).
2018-02-13 14:51:07 BetterPushback[tug.c:1194]: Error loading tug sound /Users/user/Desktop/X-Plane 11//Resources/plugins/BetterPushback/objects/tugs/AST-3F.tug/engine.wav

**Comment by jakobeng1303** [comment]
I have the same issue

**Comment by hansw1204** [comment]
I find that removing AVS gets rid of the problem for me.

**Comment by jakobeng1303** [comment]
I'll give it a try ASAP!

**Comment by Rodeo314** [comment]
A potential fix is available here: https://github.com/skiselkov/BetterPushbackC/issues/169

**Comment by skiselkov** [comment]
Please try release 0.47, it should contain a fix for the audio loading problem.

**Comment by Pilotwillem** [comment]
`Hello skiselkov
problem not solved with 0.47
2018-05-07 13:57:16 BetterPushback[acf_outline.c:198]: Error parsing acf file: property _wing/15/_Croot not found

**Comment by ahmadworks** [comment]
I had the same issue today why trying to pushback Boeing 737-900 Ultimate in macOS Hight Sierra, this is the last line in my log file:
2018-08-30 13:42:32 BetterPushback[acf_outline.c:198]: Error parsing acf file: property _wing/15/_Croot not found

**Comment by Rodeo314** [comment]
Would you care to post the full log please?

**Comment by ahmadworks** [comment]
Here is full log.
[Log7.txt](https://github.com/skiselkov/BetterPushbackC/files/2338472/Log7.txt)

**Comment by pr0f3ss** [comment]
Got a similar error log and crash while using the Zibo 737 (v3.33) right after departure:
```
2019-06-14 11:49:22 BetterPushback[acf_outline.c:198]: Error parsing acf file: property _wing/15/_Croot not found
--=={This application has crashed because of the plugin: XLua 1.0.0r1}==--
```

**Comment by skiselkov** [comment]
@pr0f3ss This crash isn't the fault of BetterPushback, it's because of XLua (as the last line clearly says). The acf parsing error is harmless, it's just BP having some trouble reading newer ACF files.

**Comment by pr0f3ss** [comment]
@skiselkov Yes sorry, you're right. I thought the cause of the crash was BP clashing with XLua, but the BP error happened a few minutes earlier than the crash actually.
